recipes/mp++/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "0.2" %}
+
+package:
+  name: mp++
```

Comment:
Hmm...so there seem to be some issues here.

First is that `+` is not permissible in a repo name. However it seems GitHub doesn't fail, it just changes `++` to `-`. Second is whether `+` is permitted in `conda` package names. Presumably `+` works ok with `conda` as the tests passed. However I'm not sure whether that is allowed in a package name or not (e.g. noted in the docs).

Currently the first issue is blocking conversion of this recipe. So we should probably decide how we want to handle that. What are your thoughts on just calling it `mppp`? Given it matches the repo name, maybe this is ok. Do you have other thoughts on naming?